jax/_src/dlpack.py

```diff
@@ -64,6 +64,14 @@ def from_dlpack(dlpack):
     gpu_backend = xla_bridge.get_backend("cuda")
   except RuntimeError:
     gpu_backend = None
+
```

Comment:
This code will probably need to change in the future. A better design would be to supply both backends to the Python code and it can use whichever it needs. Fine for now.

Reply:
@hawkinsp ok, thanks for merging!